cypress/jenkins/init.sh

```diff
@@ -3,6 +3,10 @@
 set -x
 set -e
 
+if cat /etc/os-release | grep -iq "Alpine Linux"; then
+ apk update && apk add --no-cache gcompat g++ make alpine-sdk
```

Comment:
I assume we are, but would be remiss not to ask... are we using `g++`, `make`, and `alpine-sdk` somewhere as well that are required to install here?

Reply:
Yes, we need make. We use it down the line in the script.

Reply:
I am going to test if we actually need the SDK, but the others are a must.

Reply:
@rancher-max good eye 👍🏽 the sdk isn't needed.